libraries/standard/http/include/http_client.h

```diff
@@ -363,13 +373,21 @@ typedef struct HTTPResponse
  * Upon return, #HTTPRequestHeaders_t.headersLen will be updated with the number
  * of bytes written.
  *
- * TODO: Expand documentation.
+ * Each line in the header is listed below and written in this order:
+ *     <#HTTPRequestInfo_t.method> <#HTTPRequestInfo_t.pPath> <HTTP_PROTOCOL_VERSION>
+ *     User-Agent: <HTTP_USER_AGENT_VALUE>
+ *     Host: <#HTTPRequestInfo_t.pHost>
+ *     Connection: close
+ *
+ * Note that Connection header value can be changed to keep-alive by setting
+ * the HTTP_REQUEST_KEEP_ALIVE_FLAG in #HTTPRequestInfo_t.flags.
  *
  * @param[in] pRequestHeaders Request header buffer information.
  * @param[in] pRequestInfo Initial request header configurations.
- *
- * @return #HTTP_SUCCESS if successful, an error code otherwise.
- * TODO: Update for exact error codes returned.
+ * @return One of the following:
+ * - #HTTP_SUCCESS (If successful, an error code otherwise.)
```

Comment:
(Nit) "_an error code otherwise_" is not required as the rest of the status codes call them out